LayoutTests/imported/w3c/resources/import-expectations.json

```diff
@@ -32,6 +32,10 @@
     "custom-elements": "import", 
     "density-size-correction": "import", 
     "encrypted-media": "import", 
+    "html/dom": "import", 
+    "html/syntax": "import", 
+    "html/syntax/parsing-html-fragments": "import", 
+    "html/syntax/parsing-html-fragments/innerHTML-setter-default-namespace.xhtml": "import", 
```

Comment:
These are redundant with the entries below (`web-platform-tests/html/*`), can you please undo them?

Reply:
Done.